src/Shared/Microsoft.DotNet.Scaffolding.Shared/General/StringUtil.cs

```diff
@@ -20,5 +20,10 @@ public static string ToLowerInvariantFirstChar(this string input)
             }
             return input;
         }
+
+        public static bool ContainsIgnoreCase(this string input, string value)
+        {
+            return input.ToLowerInvariant().Equals(value.ToLowerInvariant());
```

Comment:
Do we want to verify `input` and/or `value` is not null before doing `ToLowerInvariant()`? Not sure if that will throw when the string value is `null`.

Reply:
yes ty will do